demos/swift/SwiftTests/SwiftTests/AppDelegate.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AppDelegate.swift
+//  SwiftTests
+//
+//  Created by Michael Carcasole on 2017-02-22.
+//  Copyright © 2017 FOAM. All rights reserved.
```

Comment:
/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */

Reply:
Gone